src/ui/marker.js

```diff
@@ -72,6 +81,9 @@ export default class Marker extends Evented {
         this._color = options && options.color || '#3FB1CE';
         this._draggable = options && options.draggable || false;
         this._state = 'inactive';
+        this._rotation = options && options.rotation || 0;
+        this._rotationAlignment = options && options.rotationAlignment || 'auto';
+        this._pitchAlignment = options && options.pitchAlignment || this._rotationAlignment;
```

Comment:
This wouldn't match `pitchAlignment` to `rotationAlignment` if `pitchAlignment = 'auto'` and `rotationAlignment = 'map'`. This should be `options && options.pitchAlignment && options.pitchAlignment === 'auto' ? this._rotationAlignment : options.pitchAlignment`

Reply:
Fixed.